src/main/java/com/shop/controller/cars/CreateCarController.java

```diff
@@ -0,0 +1,37 @@
+package com.shop.controller.cars;
+
+import com.shop.lib.Injector;
+import com.shop.model.Car;
+import com.shop.model.Manufacturer;
+import com.shop.service.CarService;
+import com.shop.service.ManufacturerService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class CreateCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.shop");
+    private final CarService carService = (CarService) injector
+            .getInstance(CarService.class);
+    private final ManufacturerService manufacturerService = (ManufacturerService) injector
+            .getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/cars/create.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws IOException {
+        String model = req.getParameter("model");
+        Long manufacturerId = Long.valueOf(req.getParameter("manufacturerId"));
+        Manufacturer manufacturer = manufacturerService.get(manufacturerId);
+        Car car = new Car(model, manufacturer);
+        carService.create(car);
+        resp.sendRedirect(req.getContextPath() + "all");
```

Comment:
```suggestion
        resp.sendRedirect(req.getContextPath() + "/all");
```
Maybe this option is more correct? I'm not sure though